src/components/OrgPicker/OrgPicker.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * While visually similar to the USWDS Combo box component, this expandable element presents a scrollable list of links. The link at the bottom of the menu directs the user to a page listing all the organizations they can access.
+ */
+import React from 'react';
+// import { OrgObj } from '@/api/cf/cloudfoundry-types';
+import { Button } from '@/components/uswds/Button';
+import { useState } from 'react';
+import Image from 'next/image';
+import expandIcon from '@/../public/img/uswds/usa-icons/expand_less.svg';
+import collapseIcon from '@/../public/img/uswds/usa-icons/expand_more.svg';
+import { OrgPickerList } from './OrgPickerList';
+import { OrgPickerFooter } from './OrgPickerFooter';
+
+export function OrgPicker({ single }: { single: Boolean }) {
+  const [toggle, setToggle] = useState(true);
+
+  return !single ? (
+    <div className="display-block desktop:display-flex">
+      <span className="usa-label font-sans-xs padding-right-105">
+        Current organization:
+      </span>
+      <nav className="orgs-selector width-mobile border border-base-light shadow-2 font-sans-xs padding-x-105 margin-y-105">
+        <header className="display-flex padding-y-105">
+          <strong className="orgs-selector__current text-bold text-base-darker text-ellipsis margin-right-1 padding-right-1 border-right border-base-light">
+            sandbox-gsa-much-longer-name-goes-here-and-is-very-very-long
+          </strong>
+          <Button
+            unstyled
+            className="width-6"
+            onClick={() => setToggle(!toggle)}
+          >
+            <Image
+              unoptimized
+              src={toggle ? expandIcon : collapseIcon}
```

Comment:
[Non-blocking suggestion] This got me wondering: what if we just loaded one image, and rotated it with a CSS transform? Could open up some fun transition opportunities.

(Just an idea, the current version is _great_. Feel free to disregard if it’s silly!)

Reply:
Ooh I thought about it too, but wondered if it would be "too much". Rotate comin up!

Reply:
Well this rotatin’ button looks _great_, Claire. Nice work!